src/guiapi/include/gui.hpp

```diff
@@ -12,7 +12,7 @@ typedef void(gui_loop_cb_t)(void);
 
 extern gui_loop_cb_t *gui_loop_cb;
 
-extern int8_t menu_timeout_enabled;
+extern uint8_t menu_timeout_enabled;
```

Comment:
NO more extern variables!

Reply:
You reverted change. Please remove this extern.